deploy/scripts/Create-IomtWebJobsEnvironment.ps1

```diff
@@ -0,0 +1,109 @@
+<#
+.SYNOPSIS
+Creates a new IoMT FHIR Connector for Azure without using Stream Analytics
+.DESCRIPTION
+#>
+param
+(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateLength(5,12)]
+	[ValidateScript({
+        if ("$_" -cmatch "(^([a-z]|\d)+$)") {
+            return $true
+        }
+        else {
+			throw "Environment name must be lowercase and numbers"
+            return $false
+        }
+    })]
+    [string]$EnvironmentName,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('Australia East','East US','East US 2', 'West US', 'West US 2','North Central US','South Central US','Southeast Asia','North Europe','West Europe','UK West','UK South')]
+    [string]$EnvironmentLocation = "North Central US",
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('R4')]
+    [string]$FhirVersion = "R4",
+
+    [Parameter(Mandatory = $false)]
+    [string]$SourceRepository = "https://github.com/microsoft/iomt-fhir",
+  
+    [Parameter(Mandatory = $false)]
+    [string]$SourceRevision = "master",
+
+    [Parameter(Mandatory = $true)]
+    [string]$FhirServiceUrl,
+
+    [Parameter(Mandatory = $true)]
+    [string]$FhirServiceAudience,
+
+    [Parameter(Mandatory = $false)]
+    [string]$EnvironmentDeploy = $true
+)
+
+Function BuildPackage() {
+    try {
+        Push-Location $currentPath
+        cd ../../src/console/
+        dotnet restore
+        dotnet build --output $buildPath /p:DeployOnBuild=true /p:DeployTarget=Package
+    } catch {
+        throw
+    } finally {
+        Pop-Location
+    }
+}
+
+Function Deploy-WebJobs($DeviceDataWebJobName, $NormalizedDataWebJobName) {
+    try {
+        $tempPath = "$currentPath\Temp"
+        $resourceGroupName = $EnvironmentName
+        $webAppName = $EnvironmentName
+        $webJobType = "Continuous"
+
+        Clean-Path -WebJobName $DeviceDataWebJobName
+        Clean-Path -WebJobName $NormalizedDataWebJobName
+
+        $DeviceWebJobPath = "$tempPath\App_Data\jobs\$webJobType\$DeviceDataWebJobName"
+        $NormalizedWebJobPath = "$tempPath\App_Data\jobs\$webJobType\$NormalizedDataWebJobName"
+        Copy-Item "$buildPath\*" -Destination $DeviceWebJobPath -Recurse
+        Copy-Item "$buildPath\*" -Destination $NormalizedWebJobPath -Recurse
+
+        Compress-Archive -Path "$tempPath\*" -DestinationPath "$currentPath\iomtwebjobs.zip" -Force
+
+        Publish-AzWebApp -ArchivePath "$currentPath\iomtwebjobs.zip" -ResourceGroupName $resourceGroupName -Name $webAppName
+    } catch {
+        throw
+    } finally {
+        Pop-Location
+    }
+}
+
+Function Clean-Path($WebJobName) {
```

Comment:
nit: The method name uses an unapproved verb (https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.1)

Maybe Clear-Path?

Reply:
thanks - updated